packages/playwright/src/isomorphic/teleReceiver.ts

```diff
@@ -206,11 +206,10 @@ export class TeleReporterReceiver {
     let projectSuite = this._rootSuite.suites.find(suite => suite.project()!.__projectId === project.id);
     if (!projectSuite) {
       projectSuite = new TeleSuite(project.name, 'project');
+      projectSuite._project = this._parseProject(project);
       this._rootSuite.suites.push(projectSuite);
       projectSuite.parent = this._rootSuite;
     }
-    const p = this._parseProject(project);
-    projectSuite.project = () => p;
```

Comment:
I think this line was updating `project` getter to return the new project for the same suite after reload (e.g. on watch).

Reply:
oh, ui mode! Fixed.